pnpm-workspace.yaml

```diff
@@ -7,3 +7,5 @@ packages:
   - "!core/packages/http-client-csharp/**"
   - "!core/packages/http-client-java/**"
   - "!core/packages/http-client-python/**"
+  - "!core/packages/typespec-vs/**"
+  - "!core/packages/typespec-vscode/**"
```

Comment:
why is this added here? we do want to be able to build them here too, they are part of the core workspace

Reply:
Sorry, this file should not be included, reverted (I typically do this in local, as build of these plugins required specific version of .NET lib) -- appreciate any suggestion that can do e.g. "only build essential lib on pnpm build".

Reply:
Hhm, they should skip if you don't have the right requirements locally.

Can you file an issue with what you see.

I think we might want to find a more generalized way to skip things if you don't have the right items